static/js/Story.jsx

```diff
@@ -138,14 +223,17 @@ const IntroducedTextPassage = ({text, topic, afterSave, toggleSignUpModal, bodyT
     let innerContent = <ContentText html={{en: text.en, he: text.he}} overrideLanguage={overrideLanguage} bilingualOrder={["he", "en"]} />;
     const content = bodyTextIsLink ? <a href={url} style={{ textDecoration: 'none' }}>{innerContent}</a> : innerContent;
 
+
+
     return (
         <StoryFrame cls="introducedTextPassageStory">
+            <div className={"headerWithAdminButtonsContainer"}>
             <CategoryHeader type="sources" data={[topic, text]} buttonsToDisplay={["edit"]}>
                 <StoryTitleBlock en={text.descriptions?.en?.title} he={text.descriptions?.he?.title}/>
             </CategoryHeader>
-            <div className={"systemText learningPrompt"}>
-                <InterfaceText text={{"en": text.descriptions?.en?.prompt, "he": text.descriptions?.he?.prompt}} />
-            </div>
+            <ReviewStateWrapper topic={topic} text={text}></ReviewStateWrapper>
```

Comment:
Use self-closing tags here (` <ReviewStateWrapper topic={topic} text={text} />`) and for `PromptWrapper`.